cogctl/cli/profile.py

```diff
@@ -42,8 +42,9 @@ def create(state, name, host, port, secure, rest_user, rest_password):
     """
```

Comment:
Can't put it on the exact right line because nothing changed but I don't think there is a `required` param for options. We should probably prompt for input if an option is required. Maybe for another PR?